src/includes/performance-monitoring/configuration/koa.mdx

```diff
@@ -85,8 +91,8 @@ app.use(tracingMiddleWare);
 
 // usual error handler
 app.on("error", (err, ctx) => {
-  Sentry.withScope(function (scope) {
-    scope.addEventProcessor(function (event) {
+  Sentry.withScope(function(scope) {
```

Comment:
Is there a reason we're doing this with anonymous functions whereas we did the one above with arrow functions?

Reply:
No reason. I think this documentation was community contributed, but I can switch to using arrow functions.